utils/fo-installdeps

```diff
@@ -110,13 +110,13 @@ if [ $BUILDTIME ]; then
               lenny|hardy) apt-get $YesOpt install $YesOpt postgresql-server-dev-8.3;;
               squeeze) apt-get $YesOpt install $YesOpt postgresql-server-dev-8.4;;
               wheezy) apt-get $YesOpt install $YesOpt postgresql-server-dev-9.1;;
-              sid) apt-get  $YesOpt install $YesOpt postgresql-server-dev-8.4;;
               lucid|natty) apt-get $YesOpt install postgresql-server-dev-8.4;;
               isadora) apt-get $YesOpt install postgresql-server-dev-8.4;;
               precise|oneiric|quantal|raring|saucy) apt-get $YesOpt install postgresql-server-dev-9.1;;
               trusty) apt-get $YesOpt install postgresql-server-dev-9.3;;
               jessie) apt-get $YesOpt install postgresql-server-dev-9.4;;
-              xenial) apt-get $YesOpt install postgresql-server-dev-9.5;;
+              xenial|yakkety) apt-get $YesOpt install postgresql-server-dev-9.5;;
+              stretch|buster|sid|zesty|artful) apt-get $YesOpt install postgresql-server-dev-9.6;;
```

Comment:
> to be clear, the point i raised was that ubuntu support is actually more work to maintain than debian because it is guaranteed to requires a routine update to these install files every 6 months - even if no dependencies change, a new ubuntu entry would need to be added every 6 months - that would not be the case for debian - unless dependencies change, a new debian entry would not need to be added until the next debian LTS version in 2-3 years - if every ubuntu version is to be supported, then keeping 'buster' and 'sid' will be exactly zero additional work because whatever dependency versions ubuntu requires, they take from what is current in debian testing (currently buster)

But Ubuntu will at least not break in the future. Unless someone of the developers uses buster or sid, we will not recognize when the dependencies break on these systems.

And AFAIK we can not test the dependencies with travis (unless we set up docker files for that)?